.ts_pre_commit_config.yaml

```diff
@@ -0,0 +1,10 @@
+black: true
+check-xml: true
+check-yaml: true
+clang-format: false
+flake8: true
+format-xmllint: false
+isort: true
+mypy: false
```

Comment:
Are you sure you want to set this to `false`? I see you include typing info (almost?) everywhere.

Reply:
Good point, I'll make the change to use mypy :+1:.